config/software/nghttp2.rb

```diff
@@ -0,0 +1,33 @@
+name "nghttp2"
+default_version "1.12.0"
+
+dependency "openssl"
+dependency "cython"
+
+source :url => "https://github.com/nghttp2/nghttp2/releases/download/v1.12.0/nghttp2-#{version}.tar.gz",
+       :md5 => "04235f1d7170a2efce535068319180a1"
+
+relative_path "nghttp2-#{version}"
+
+env = {
+  "CYTHON" => "#{install_dir}/embedded/bin/cython",
+  "PYTHON" => "#{install_dir}/embedded/bin/python2.7",
+  "PYTHON_CPPFLAGS" => "-I#{install_dir}/embedded/include/python2.7",
+  "PYTHON_LDFLAGS" => "-L#{install_dir}/embedded/lib -lpython2.7",
+  "OPENSSL_CFLAGS" => "-I#{install_dir}/embedded/include/openssl",
+  "OPENSSL_LIBS" => "-L#{install_dir}/embedded/lib"
+}
+
+build do
+  command [
+    "./configure",
+    "--enable-python-bindings",
+    "--disable-app",
+    "--disable-examples",
+    "--disable-hpack-tools",
+    "--prefix=#{install_dir}/embedded",
+    ""
+  ].join(" "), :env => env
```

Comment:
the `configure` step is failing on my `deb-x64` docker container, looks like it's missing `LDFLAGS`:
```
    configure: WARNING:
  Could not link test program to Python. Maybe the main Python library has been
  installed in some non-standard library path. If so, pass it to configure,
  via the LDFLAGS environment variable.
  Example: ./configure LDFLAGS="-L/usr/non-standard-path/python/lib"
  ============================================================================
   ERROR!
   You probably have to install the development version of the Python package
   for your distribution.  The exact name of this package varies among them.
  ============================================================================

configure: WARNING: No package 'cunit' found
configure: error: python bindings were requested (--enable-python-bindings) but dependencies are not met.
```

have you tried setting the standard flags for `env` with `with_standard_compiler_flags()` and then adding your specific flags? See https://github.com/DataDog/omnibus-software/blob/5.8.5/config/software/libpcap.rb#L32-L35 for an example